### PR TITLE
Minor graph update, shift gauge graph up to be centered with two outer pie graphs..

### PR DESCRIPTION
**Pull request by pir8radio**
minor change to shift up center gauge to match pie graph centering..   minor eye sore fix..  :-)

### CODE REVIEW

**Comment by frankTheTank72** [comment]
okay for me

File: scan/templates/home/index.html [hunk 1]
```diff
@@ -80,11 +80,17 @@ <h6 class="text-uppercase text-secondary mb-0"><i class="fas fa-project-diagram
...
+            <div class="media align-items-center mt-n2" Style="max-height: 120px;">
+              <!--<canvas style="max-height: 135px; width: 100vw; display: block;" id="myChart"></canvas>-->
+            </div>
+          </div>
         </div>
       </div>
     </div>
   </div>
 
+<div id="update_Latest">
```
Comment (jjos2372) [inline scan/templates/home/index.html:93]: Can this be called "update-latest"? (mixing CamelCase with underline looks bad)

File: scan/templates/txs_pending/list.html [hunk 2]
```diff
@@ -23,5 +24,12 @@ <h5 class="p-2">Pending Transactions</h5>
       </div>
     </div>
   </div>
+</div>
+
+<script>
+  setInterval(function () {
+    $('#update_pending').load(location.href + ' #update_pending');
+  }, 10000); //update every X seconds
```
Comment (jjos2372) [inline scan/templates/txs_pending/list.html:32]: miliseconds?

File: scan/templates/home/index.html [hunk 3]
```diff
@@ -121,12 +127,62 @@ <h5 class="card-header-title">Pending Transactions</h5>
...
+    title: {display:true, text:'TRANSACTIONS PER-HOUR', font:{family: "'Montserrat', sans-serif", weight: 500,}},
+             }
+         }
+});
+</script>-->
+
+<script>
+  setInterval(function () {
+    $('#update_Latest').load(location.href + ' #update_Latest');
+  }, 60000); //update every X seconds
```
Comment (jjos2372) [inline scan/templates/home/index.html:185]: miliseconds I guess

**Review by jjos2372 — CHANGES_REQUESTED** [review]
Just some small recommendations.

File: scan/templates/home/index.html (hunk 1 above)
Comment (pir8radio) [reply scan/templates/home/index.html:93]: yes... ill make the change.

File: scan/templates/txs_pending/list.html (hunk 2 above)
Comment (pir8radio) [reply scan/templates/txs_pending/list.html:32]: lol, yes..  just notes for us... ill change description to ms.

File: scan/templates/home/index.html (hunk 3 above)
Comment (pir8radio) [reply scan/templates/home/index.html:185]: yes...  ill update all of these on my end..

**Pull request merged**